.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Can you check if this matches the one in Graasp Dssktop?

Reply:
it doesn't. specifically: desktop only uses `eslint-config-airbnb`, `eslint-config-prettier`, and `eslint-plugin-mocha`, while graasp-research is missing the latter but has the first 2 *plus* `eslint-plugin-import`, `eslint-plugin-jsx-a11y`, `eslint-plugin-react` (and corresponding amends to `.eslintrc`). since this is a react app (even if on desktop) I assumed we should follow the pattern of graasp-research. but i can amend it to be exactly like graasp-desktop. @juancarlosfarah let me know!

Reply:
Indeed it should have the React part too. Let's see what @pyphilia says.

Reply:
I think it should work like this, you may add `eslint-plugin-mocha` as well since you included it in `env` already.

Reply:
done!